projects/backend/__tests__/application.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { Request, Response } from 'express'
+import { EditionsBackendControllers, createApp } from '../application'
+import chai from 'chai'
+import chaiHttp from 'chai-http'
+import * as http from 'http'
+
+// Configure chai
+chai.use(chaiHttp)
+chai.should()
+
+const stub = (req: Request, res: Response) => {
+    console.log(`endpoint: ${req.path} called`)
+    res.sendStatus(200)
+}
+
+const testStubControllers: EditionsBackendControllers = {
+    issuesSummaryController: stub,
+    issueController: stub,
+    frontController: stub,
+    imageController: stub,
+    imageColourController: stub,
+}
+
+const previewApp = createApp(testStubControllers, true)
+const versionedApp = createApp(testStubControllers, false)
```

Comment:
Is it possible to create dependencies inside describe blocks? If so this would help prevent the wrong app being using in the wrong test. Alternatively we could split this into two tests.

Reply:
done